lib/proxymode.pm

```diff
@@ -127,4 +127,75 @@ sub redirect_serial {
     select_console "root-console";
     $self->reset_curr_serialdev();
 }
+
+sub get_consoledev {
```

Comment:
yes, you are right, i wrap this function is used to flexibly calling under this project.